jdi-light/src/main/java/com/epam/jdi/light/elements/complex/dropdown/DropdownExpand.java

```diff
@@ -112,5 +116,4 @@ public boolean isDisplayed() {
     public boolean isEnabled() {
         return value().isEnabled();
     }
-
 }
```

Comment:
Please ensure you have a blank line at the end of all files you modified

Reply:
have a blank line in the end of file